packages/cursorless-engine/src/util/allocateHats/allocateHats.ts

```diff
@@ -135,19 +143,27 @@ function getTokenOldHatMap(oldTokenHats: readonly TokenHat[]) {
 function getTokenRemainingHatCandidates(
   tokenGraphemeSplitter: TokenGraphemeSplitter,
   token: Token,
-  availableGraphemeStyles: DefaultMap<string, HatStyleMap>,
+  availableGraphemeStyles: DefaultMap<string, HatStyleName[]>,
+  allHatStyles: HatStyleMap,
```

Comment:
why not call this `enabledHatStyles` like we do in the calling function?

Reply:
done